docker/Dockerfile

```diff
@@ -1,5 +1,8 @@
 FROM debian:9.5-slim
 
+# Default google-fluentd at version 1.6.0-1.
+ARG repo_suffix=20181011-3
```

Comment:
`ARG` names tend to be uppercase (e.g., [here](https://docs.docker.com/engine/reference/builder/#understand-how-arg-and-from-interact)). Should we follow that pattern?